### DISCUSSION TITLE
How do I take data from a form and use it in a automation.

### DISCUSSION THREAD

**Discussion by adamsanclemente** [Budibase/budibase#3893] [Help] [answered]
I want to have an automation where it sends a message to a discord server once a form has been filled out, simple enough.. but I can't figure out how to get the data from the form in the automation so I can include some of the info in the discord message. Any help appreciated.

**Comment by shogunpurple** [comment, marked as answer]
Hi @PixelParallax - the easiest way to do this is by adding a `Row Created` automation, which will trigger once you save the data from your form.

Here's a demo showing how you would do this in the automations section of the builder. Hope this helps!

https://user-images.githubusercontent.com/11256663/147742274-c63d50b0-67fd-4188-83a0-2935a274c58f.mov

**Reply by adamsanclemente** [reply]
Thank you! I appreciate the video. Helped.
